### DISCUSSION TITLE
How to publish to maven local?

### DISCUSSION THREAD

**Discussion by Mehequanna** [sceneview/sceneform-android#429] [Q&A] [answered]
Hello! 

I love the work you are doing here! I just found this work in the last week or so. So I'm fairly new. I wanted to play around with this code and maybe make updates. Is there a way to publish changes to my Local Maven repository so I can test the code I'm updating?

Thank you! :)

**Comment by grassydragon** [comment, marked as answer]
Hi!
Yes, you can publish a release from Android Studio to your local Maven repository. However, you may also want to check out the SceneView project that gets our full attention now: https://github.com/SceneView/sceneview-android

**Reply by Mehequanna** [reply]
Thanks @grassydragon ! I have been looking at Sceneview. It's looking great. I have a more immediate need for Sceneform which is why I wanted to start here. I'll keep an eye on Sceneview for sure!

**Reply by Mehequanna** [reply]
@grassydragon I'm not seeing a way to publish to maven local via Android Studio. I know you are busy, but if you have a moment and can send me some documentation or an example that would be awesome :) If not, I don't appreciate the time you have given!

**Reply by grassydragon** [reply]
In SceneView we have [gradle-maven-publish-plugin](https://github.com/vanniktech/gradle-maven-publish-plugin) enabled that adds a `publishMavenPublicationToMavenLocal` task: https://github.com/SceneView/sceneview-android/blob/bd71e8a431353d682cc8c1e69e7702d0537e93fe/build.gradle#L11
You can try uncommenting the plugin usages in Sceneform to see if the publishing task works: https://github.com/SceneView/sceneform-android/blob/master/sceneform/build.gradle#L4

**Reply by Mehequanna** [reply]
I'll give that a try, thanks!

**Reply by Mehequanna** [reply]
@grassydragon it worked! Thank you again :)
